subprojects/internal-performance-testing/src/main/groovy/org/gradle/performance/generator/JavaTestProjectGenerator.groovy

```diff
@@ -85,6 +86,13 @@ enum JavaTestProjectGenerator {
         .withDsl(KOTLIN)
         .create()),
 
+    LARGE_EMPTY_MULTI_PROJECT_DECLARATIVE_DSL(new TestProjectGeneratorConfigurationBuilder("largeEmptyMultiProjectDeclarativeDsl", "largeEmptyMultiProject")
+        .withSubProjects(250)
```

Comment:
❌ Let's make this 500 just like the other large multi project templates so we can compare results more easily